docs/radar.js

```diff
@@ -44,6 +44,7 @@ function radar_visualization(config) {
   config.title_offset = config.title_offset || { x: -675, y: -420 };
   config.footer_offset = config.footer_offset || { x: -155, y: 450 };
   config.legend_column_width = config.legend_column_width || 140
+  config.line_height = 10
```

Comment:
```suggestion
  config.line_height = config.line_height || 10
```

Reply:
Also, can you rename the variable to `legend_line_height` to be more accurate in what it controls?

Reply:
Done 🚀